aom/src/main/java/com/nedap/archie/aom/primitives/CTerminologyCode.java

```diff
@@ -186,6 +195,29 @@ public List<String> getValueSetExpanded() {
         }
         return result;
     }
+    
+    private List<String> getOpenEHRValueSetExpanded() {
+        List<String> atCodes = getValueSetExpanded();
```

Comment:
It is also possible to bind a value set to an ac-code, and to bind that to an external value set, the same way as binding an at-code. I think that is not currently possible with openEHR-codes, since no way to address the individual groups within the terminology exists.